build.gradle

```diff
@@ -1,6 +1,7 @@
 buildscript {
 	repositories {
         jcenter()
+        mavenCentral()
```

Comment:
Are these changes the solution to the build issue that you were facing with building the docker image?

Reply:
Yes.